src/data/nat/prime.lean

```diff
@@ -3,9 +3,29 @@ Copyright (c) 2015 Microsoft Corporation. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Leonardo de Moura, Jeremy Avigad, Mario Carneiro
 
-Prime numbers.
 -/
-import data.nat.sqrt data.nat.gcd data.list.basic data.list.perm tactic.wlog
+import data.nat.sqrt data.nat.gcd data.list.basic data.list.perm
+import algebra.group_power
+import tactic.wlog
+
+/-!
+# Prime numbers
+
+This file deals with prime numbers: natural numbers p ≥ 2 whose only divisors are `p` and `1`.
```

Comment:
Missing backticks around p ≥ 2. Also below in `n ≠ 1`. And in the docstring of `exists_infinite_primes`.

Reply:
Fixed.